kombu/connection.py

```diff
@@ -815,8 +815,18 @@ def default_channel(self):
             a connection is passed instead of a channel, to functions that
             require a channel.
         """
+        ensure_connection_options = {}
+        if self.transport_options:
+            options_keys = [
+                'max_retries',
+                'interval_start', 'interval_step', 'interval_max',
+            ]
+            for k in options_keys:
```

Comment:
There's still no need for a loop here.
Please hardcode the values as suggested in the last review.